### PR TITLE
fix the unit tests

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log

- set the correct ```.travis.yml```.
- set the different PHPUnit version for the different PHP versions in ```composer.json```.
- fix the incorrect unit tests in the previous broken [tests](https://github.com/vimeo/vimeo.php/commit/932123dfcc0d47245473ade5033f7a9e1a0c4bae).

Thanks.

### CODE REVIEW

File: tests/Vimeo/VimeoTest.php [hunk 1]
```diff
@@ -176,7 +176,7 @@ public function testUploadWithFakeMachineIdShouldReturnVimeoRequestException()
         $vimeo = new Vimeo($this->clientId, $this->clientSecret);
 
         // Act
-        $result = $vimeo->upload(__DIR__.'/../../composer.json');
+        $result = $vimeo->upload(__DIR__.'/../../composer.json', array('fake_machine_id'));
```
Comment (erunion) [inline tests/Vimeo/VimeoTest.php]: We don't support machine ID's anymore in the API, so you can remove these changes to `VimeoTest`.

**Comment by peter279k** [comment]
Hi @erunion, thank you for your reply.
I have removed the ```fake_machine_id``` param for the upload API test.
And I also chenage into method named ```testUploadWithInvalidParamShouldReturnVimeoRequestException```.
If having any request chnages, please feel free to send the any suggestions.

Thanks.

File: tests/Vimeo/VimeoTest.php [hunk 2]
```diff
@@ -214,4 +214,40 @@ public function testUploadTexttrackWithNonExistedFile()
         // Act
         $result = $vimeo->uploadTexttrack('https://vimeo.com/241711006', './the_file_is_invalid', 'fake_track_type', 'zh_TW');
     }
+
+    /**
+     * @expectedException Vimeo\Exceptions\VimeoRequestException
+     */
+    public function testReplaceWithVideoUriAndMachineIdShouldReturnVimeoRequestException()
```
Comment (erunion) [inline tests/Vimeo/VimeoTest.php]: Please remove these other three machine ID tests, or change them to invalid param tests like you did with `testUploadWithInvalidParamShouldReturnVimeoRequestException`.

**Comment by erunion** [comment]
Awesome, thank you!

**Pull request merged**